ui/src/App.tsx

```diff
@@ -131,6 +132,7 @@ function App() {
           <Route path="/issuers" element={<TrustedIssuersPage service={blockchainService} />} />
           <Route path="/identities" element={<IdentitiesPage service={blockchainService} />} />
           <Route path="/claims" element={<ClaimsPage service={blockchainService} />} />
+		  <Route path="/identities/id" element={<DigitalIdentityDetailView />} />
```

Comment:
the path here should be identites/{selectedId}

Reply:
Done